packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.controller.js

```diff
@@ -178,6 +178,23 @@ export default class TelecomPackMigrationConfirmCtrl {
     return true;
   }
 
+  getMeeting() {
+    const day = moment(
+      this.process.selectedOffer.meetingSlots.slot.startDate,
+    ).format('DD/MM/YYYY');
+    const start = moment(
+      this.process.selectedOffer.meetingSlots.slot.startDate,
+    ).format('HH:mm');
+    const end = moment(
+      this.process.selectedOffer.meetingSlots.slot.endDate,
+    ).format('HH:mm');
```

Comment:
For better readabilty

```suggestion
    const { startDate, endDate } =  this.process.selectedOffer.meetingSlots.slot;
    const day = moment(startDate).format('DD/MM/YYYY');
    const start = moment(startDate).format('HH:mm');
    const end = moment(endDate).format('HH:mm');
```